app/assets/javascripts/app/core/mobile_detect.js

```diff
@@ -0,0 +1,19 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('council.core')
+    .factory('mobileDetect', mobileDetect);
```

Comment:
Please read the comments with attention, I already said that's this is not a factory, it's a service, please read [this](https://www.google.pt/search?q=factory+programming&oq=factory+programming&aqs=chrome..69i57.5804j0j7&sourceid=chrome&es_sm=91&ie=UTF-8) . Also, I already commented that `mobileDetect` is not a good name.